packages/web-app-files/tests/unit/helpers/breadcrumbs.spec.ts

```diff
@@ -14,15 +18,18 @@ describe('builds an array of breadcrumbitems', () => {
 
   it('from an array of breadcrumbitems', () => {
     const initialBreadCrumbs = [{ text: 'Foo' }, { text: 'Bar' }]
-    const breadCrumbsFromPath = breadcrumbsFromPath({ path: '/app/foo/bar?all=500' }, '/bar')
-    const result = concatBreadcrumbs(...initialBreadCrumbs, breadCrumbsFromPath)
+    const breadCrumbsFromPath = breadcrumbsFromPath(
+      { path: '/app/foo/bar?all=500' } as RouteLocation,
+      '/bar'
+    )
+    const result = concatBreadcrumbs(...initialBreadCrumbs, ...breadCrumbsFromPath)
     expect(result[0]).toMatchObject({ text: 'Foo' })
     expect(result[1]).toMatchObject({ text: 'Bar' })
     expect(JSON.stringify(result[2])).toEqual(
       JSON.stringify({
-        allowContextActions: undefined,
-        onClick: () => jest.fn(() => {}),
-        text: undefined
+        allowContextActions: true,
+        text: 'bar',
+        onClick: () => undefined
       })
     )
   })
```

Comment:
IMHO this test was completely broken as `breadCrumbsFromPath` was not spread into `concatBreadcrumbs` which just accepts a single rest args parameter.